app/models/katello/content_view_docker_filter.rb

```diff
@@ -21,6 +21,15 @@ def generate_clauses(repo)
       { "_id" => { "$in" => manifest_tags } } unless manifest_tags.empty?
     end
 
+    def get_pulp_ids(repo)
+      manifest_tags = []
+      self.docker_rules.each do |rule|
+        manifest_tags.concat(query_manifests(repo, rule))
+      end
+
+      manifest_tags
+    end
```

Comment:
we usually try to avoid 'get_' methods, maybe just name it 'pulp_ids'?  or 'content_unit_pulp_ids' to be more specific/